core-web/package.json

```diff
@@ -221,7 +221,6 @@
         "fake-indexeddb": "^5.0.2",
         "fetch-mock": "^9.11.0",
         "flatpickr": "^4.5.7",
-        "gh-pages": "^3.1.0",
```

Comment:
Delete where this is being used.

Reply:
The action too?

Reply:
I would say so... if not, why are you deleting this?